packages/rocketchat-channel-settings/client/views/channelSettings.coffee

```diff
@@ -119,13 +119,13 @@ Template.channelSettings.onCreated ->
 		room = ChatRoom.findOne @data?.rid
 		switch @editing.get()
 			when 'roomName'
-				if $('input[name=roomName]').val() is room.name
+				if $('input[name=roomName]').val().trim() is room.name
```

Comment:
@tgxn can we make $('input[name=roomName]').val().trim() a var, so that we don't have to duplicate what ever we do three times? It improves readability and more maintainable.